sdks/go/sdk.go

```diff
@@ -42,6 +42,14 @@ type SDK struct {
 	alpha  *Alpha
 }
 
+// ErrorLog is a function to log the error.
+type ErrorLog func(string, error)
+
+// Logger is a pluggable function that outputs the error message to standard error.
+var Logger ErrorLog = func(msg string, err error) {
+	fmt.Fprintf(os.Stderr, "%s: %s\n", msg, err)
```

Comment:
```suggestion
  if gs == nil {
    return
  }
	fmt.Fprintf(os.Stderr, "%s: %s\n", msg, err)
```

Since @clairesc 's comment is very good point!

Reply:
can I add the `gs == nil` check here? WDYT?
log := func(gs *sdk.GameServer, msg string, err error) {
    if gs == nil || gs.ObjectMeta.DeletionTimestamp == 0 {
        return
    }
    Logger(msg, err)
}

Reply:
Yeah that's way better.